.nvmrc

```
@@ -1 +1 @@
-10
+14
```

```suggestion
14.18
```

To match with the version we're using in CircleCI